test/TensorFlowRuntime/dynamic_compilation_tensor_group.swift

```diff
@@ -112,6 +114,26 @@ TensorGroupTest.testAllBackends("input, loadable") {
   expectEqual(ShapedArray(shape: [2, 2], scalars: [1, 2, 3, 4]), packed.array)
 }
 
+@inline(never)
+func some_tf_op(n : Int) {
+  var arr: [Tensor<Float>] = []
+  var arr_exp: [Float] = []
+  for i in 0..<n {
+    arr.append(Tensor<Float>([Float(i), 2, 3]))
+    arr_exp.insert(contentsOf: [Float(i), 2, 3], at: arr_exp.count)
+  }
+
+  let ret: TensorHandle<Float> = #tfop("Pack",
+    [arr], T$dtype: Float.tensorFlowDataType, axis: 0)
+  let t: Tensor<Float> = Tensor<Float>(shape: [Int32(n), 3], scalars: arr_exp)
+  expectEqual(Tensor<Float>(handle: ret).array, t.array)
```

Comment:
Our convention is to have the expected value first.

Also you can simplify this by making the expected ShapedArray directly with `ShapedArray(shape: scalars:)` instead of making a tensor and converting it to array.